modules/pull.lua

```diff
@@ -854,17 +854,20 @@ function Module:CheckGroupForPull(classes, resourceStartPct, resourceStopPct, ca
 
         if member and member.ID() > 0 then
             if not classes or classes:contains(member.Class.ShortName()) then
-                local resourcePct = self.TempSettings.PullState == PullStates.PULL_GROUPWATCH_WAIT and resourceStopPct or resourceStartPct
+                local resourcePct = self.TempSettings.PullState == PullStates.PULL_GROUPWATCH_WAIT and resourceStartPct or resourceStopPct
                 if member.PctHPs() < resourcePct then
                     RGMercUtils.PrintGroupMessage("%s is low on hp - Holding pulls!", member.CleanName())
+                    RGMercsLogger.log_warn("\arMember is low on hp - \ayHolding pulls!\ax\ag ResourcePCT:\ax \at%s \aoStopPct: \at%s \ayStartPct: \at%s \aoPullState: \at%s",resourcePct,resourceStopPct,resourceStartPct, self.TempSettings.PullState)

```

Comment:
Please use %d (or %g if lua supports that) for printing numbers and not %s which is for strings.